azurelinuxagent/common/event.py

```diff
@@ -72,6 +72,7 @@ class WALAEventOperation:  # pylint: disable=no-init
     ArtifactsProfileBlob = "ArtifactsProfileBlob"
     CGroupsCleanUp = "CGroupsCleanUp"
     CGroupsDebug = "CGroupsDebug"
+    CGroupsDisabled = "CGroupsDisable"
```

Comment:
NIT: Shouldn't it be `"CGroupsDisable**d**"` instead?

Reply:
You may need to refresh... a second commit fixed this typo

Reply:
I dont see it?  Did you push your changes yet haha?

Reply:
ah, you mean the string value, I'll fix that